argparsing.py

```diff
@@ -106,6 +112,14 @@ def error(self, message):
     choices=["csv", "json"],
 )
 
+parser.add_argument(
+    "--name-server",
```

Comment:
perhaps call it `resolver` to be more specific?

Reply:
done